XCUITests/ActivityStreamTest.swift

```diff
@@ -143,12 +143,16 @@ class ActivityStreamTest: BaseTestCase {
         navigator.performAction(Action.PinToTopSitesPAM)
         // Workaround to have visited website in top sites
         navigator.performAction(Action.AcceptRemovingAllTabs)
+        navigator.performAction(Action.CloseURLBarOpen)
         navigator.performAction(Action.OpenNewTabFromTabTray)
 
         waitForExistence(app.collectionViews.cells[newTopSite["topSiteLabel"]!])
         XCTAssertTrue(app.collectionViews.cells[newTopSite["topSiteLabel"]!].exists)
         checkNumberOfExpectedTopSites(numberOfExpectedTopSites: 6)
 
+        navigator.goto(NewTabScreen)
+        app.buttons["urlBar-cancel"].tap()
```

Comment:
@SimonBasca I see the problem. It tries to go to settings from a new tab, so you can fix that by adding these two lines in L153 and 154 instead of those two you have: 
```
navigator.performAction(Action.CloseURLBarOpen)
navigator.nowAt(NewTabScreen)
```
to kind of tell the app that you are already in a new tab and you only need to open settings. I tried that locally and it seems to be working